index.html

```diff
@@ -197,38 +245,41 @@ <h3 class="text1">Language</h3>
     </div>
  </section>
  <footer class="footer">
-<p class="subject">
-    I'm always<br> interested in hearing<br> about new projects,<br> so if you'd like to<br> chat please get in
-</p>
-<form action="https://formspree.io/f/mwkzpjrj" method="post" class="form">
-<div class="full-name">
-    <input type="text" name="name" id="name" maxlength="30" required placeholder="Full name">
-</div>
-<hr class="hor">
-<div class="email">
-    <input type="email" name="email" id="email" required placeholder="Email address">
-</div>
-<hr class="hor">
-<div class="textarea">
-    <textarea name="message" id="message" cols="45" rows="6" required maxlength="500">Enter text here</textarea>
-</div>
-<div class="button-cont">
-    <button type="submit"  class="buttonl">Get in touch</button>
+    <div class="last-div">
+        <div class="try">
+            <p class="subject">
+                I'm always interested<br> in hearing about new<br> projects, so if you'd<br> like to chat please<br> get in
+            </p>
+        </div>        
+        <form action="https://formspree.io/f/mwkzpjrj" method="post" class="form">
+            <div class="full-name">
+                <input type="text" name="name" id="name" maxlength="30" required placeholder="Full name">
+            </div>
+            <hr class="hor">
+            <div class="email">
+                <input type="email" name="email" id="email" required placeholder="Email address">
+            </div>
+            <hr class="hor">
+            <div class="textarea">
+                <textarea name="message" id="message" cols="40" rows="6" required maxlength="500">Enter text here</textarea>
+            </div>
+            <div class="button-cont">
+                <button type="submit"  class="buttonl">Get in touch</button>
```

Comment:
- [ ] kindly add the hover , focus, and click styling to all of your buttons following the Figma design 

![image](https://user-images.githubusercontent.com/89282221/188263110-08b44e5b-5824-4c7e-9209-d1d6c29b6ee2.png)